build.gradle

```diff
@@ -89,7 +89,7 @@ project.ext.externalDependency = [
     'elasticSearchRest': 'org.elasticsearch.client:elasticsearch-rest-high-level-client:' + elasticsearchVersion,
     'elasticSearchTransport': 'org.elasticsearch.client:transport:' + elasticsearchVersion,
     'findbugsAnnotations': 'com.google.code.findbugs:annotations:3.0.1',
-    'graphqlJava': 'com.graphql-java:graphql-java:' + graphQLJavaVersion,
+    'graphqlJava': 'com.graphql-java:graphql-java:19.4',
```

Comment:
The version here should be incremented above from `ext.graphQLJavaVersion = '19.0'` to the 19.4 version.